docs/_tooling/extensions/score_metamodel/metamodel.yaml

```diff
@@ -132,6 +132,9 @@ needs_types:
       status: "^(valid|draft|invalid)$"
     optional_options:
       realizes: "^wp__.+$"
+      # req-Id: gd_req__req__structure
+      # req-Id: gd_req__requirements_attr_description
+      # req-Id: gd_req__req__linkage
```

Comment:
I don't see where those 3 are implemented? Can you move them, so the implementation is always on the next line, in the next group, etc